internal/backends/mcu/pico_st7789.rs

```diff
@@ -56,27 +54,28 @@ const SPI_ST7789VW_MAX_FREQ: embedded_time::rate::Hertz = embedded_time::rate::H
 
 const DISPLAY_SIZE: PhysicalSize = PhysicalSize::new(320, 240);
 
+/// The Pixel type of the backing store
+pub type TargetPixel = embedded_graphics::pixelcolor::Rgb565;
+
 pub fn init() {
     unsafe { ALLOCATOR.init(&mut HEAP as *const u8 as usize, core::mem::size_of_val(&HEAP)) }
-    i_slint_core::platform::set_platform_abstraction(alloc::boxed::Box::new(PicoBackend))
+    i_slint_core::platform::set_platform_abstraction(Box::new(PicoBackend::default()))
         .expect("backend already initialized");
 }
 
-thread_local! { static WINDOW: RefCell<Option<Rc<PicoWindow>>> = RefCell::new(None) }
-
-struct PicoBackend;
+#[derive(Default)]
+struct PicoBackend {
+    window: RefCell<Option<Rc<PicoWindow>>>,
```

Comment:
I think that this could also just be a `Rc<PicoWindow>`. I don't think the creation of the window requires anything special or is timing dependent.

Reply:
that's true.
But i still like that create_window actually creates the window